City-of-Helsinki/drupal-hdbt-admin
# UHF-3615: Fix group edit header

# Groups for general upper secondary schools [UHF-3615](https://helsinkisolutionoffice.atlassian.net/browse/UHF-3615)
Fix group edit page header.

## What was done
* Add new template that overrides the default gin-template and a small css fix.

## How to install
* Check instructions from https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus/pull/91

## How to test
* Check instructions from https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus/pull/91

## Other PRs
* https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus/pull/91
* https://github.com/City-of-Helsinki/drupal-hdbt/pull/258
